samples/unit-tests/maps/mapbubble-linewidth/demo.js

```diff
@@ -1,41 +1,55 @@
-// This should maybe be a visual test
-QUnit.test('MapBubble with LineWidth', function (assert) {
-    var chart = Highcharts.mapChart('container', {
-        series: [
-            {
-                mapData: Highcharts.maps['countries/gb/gb-all']
-            },
-            {
-                type: 'mapbubble',
-                lineWidth: 2,
-                data: [
-                    {
-                        lat: 51.507222,
-                        lon: -0.1275,
-                        z: 3
-                    },
-                    {
-                        lat: 52.483056,
-                        lon: -1.893611,
-                        z: 4
-                    },
-                    {
-                        x: 1600,
-                        y: -3500,
-                        z: 3
-                    },
-                    {
-                        x: 2800,
-                        y: -3800,
-                        z: 1
-                    }
-                ]
-            }
-        ]
-    });
+QUnit.test('MapBubble', function (assert) {
+    const chart = Highcharts.mapChart('container', {
+            series: [
+                {
+                    mapData: Highcharts.maps['countries/gb/gb-all']
+                },
+                {
+                    type: 'mapbubble',
+                    lineWidth: 2,
+                    data: [
+                        {
+                            lat: 51.507222,
+                            lon: -0.1275,
+                            z: 3
+                        },
+                        {
+                            lat: 52.483056,
+                            lon: -1.893611,
+                            z: 4
+                        },
+                        {
+                            x: 1600,
+                            y: -3500,
+                            z: 3
+                        },
+                        {
+                            x: 2800,
+                            y: -3800,
+                            z: 1
+                        }
+                    ]
+                }
+            ]
+        }),
+        controller = new TestController(chart);
+
     assert.strictEqual(
         chart.series[1].graph['stroke-width'],
         2,
-        'Points have stroke width'
+        'MapBubble with linewidth- points should have stroke width.'
+    );
+
+    chart.update({
+        tooltip: {
+            shared: true
+        }
+    });
+    // Hover over the chart.
+    controller.mouseMove(200, 200);
```

Comment:
Instead of using TestController, you can simulate hovering with
```js
chart.pointer.runPointActions({ chartX: 400, chartY: 300 });
```

TestController is good, it's your call what you choose. IMHO my way is a little bit easier and faster